src/Controller/FightController.php

```diff
@@ -49,7 +49,8 @@ public function pick()
             }
         }
         if (isset($_SESSION['player1']) && isset($_SESSION['player2'])) {
-            header('Location: /fight');
+            new FightController();
```

Comment:
looks like `new FightController();` is useless here ;)

Reply:
oh yes mb